chocolatey/datahelix/datahelix.nuspec

```diff
@@ -0,0 +1,29 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- Test your packages in a test environment: https://github.com/chocolatey/chocolatey-test-environment -->
+
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>datahelix</id>
+    <version>0.0.0</version>
```

Comment:
should this match our latest release?

Reply:
This is supplanted with the latest release number during the automatic update process